### DISCUSSION TITLE
Mac Silicon Custom Nodes issue

### DISCUSSION THREAD

**Discussion by erikjlundy** [community/community#129019] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi there, I'm having an issue using an Animate Diff workflow that points at one of your custom nodes.  I was hoping you might have some feedback to help.  I'm on a Mac book with an M3Pro chip and 18 gigs of RAM. I've tried installing the custom nodes manually, via the manager and via git clone. and I keep getting the following error when loading the workflow.  Any help is completely appreciated .  When loading the graph, the following node types were not found:
JWInteger
Nodes that have failed to load will show as red on the graph.

**Comment by ebndev** [comment]
Hi @erikjlundy, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? 

That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by erikjlundy** [comment]
I'm new to this, so thank you very much for pointing me in the right
direction.
Thank you,
Erik
